.gitignore

```diff
@@ -6,6 +6,7 @@ bin-release/
 
 # Other files and folders
 .settings/
+test.md
 
 # Executables
 *.swf
```

Comment:
- Please include `node_modules` in your `gitignore` file, it might be accidentally added while setting up linters 👍 👍

Reply:
Done

Reply:
> * Please include `node_modules` in your `gitignore` file, it might be accidentally added while setting up linters 👍 👍

@mckent05 Please can you check if I managed to fix the issues you highlighted before I can submit another code review thanks.